docs/developer-center/guides/01-Quickstart.md

```diff
@@ -71,15 +71,15 @@ cc = CartoContext(
 
 ```python
 from cartoframes import Credentials, CartoContext
-creds = Credentials(user='<your_user_name>', key='<your_api_key>')
+creds = Credentials(username='<your_user_name>', key='<your_api_key>')
```

Comment:
Thanks, great catch! Once we merge this into master, we'll need to merge master into [this PR](https://github.com/CartoDB/cartoframes/pull/558/files#diff-e023f6411ac78a05e315d320aca2d2b1).

Reply:
I was trying some examples and I found this problem. I did know you had created another one, let me move the commit to your initial PR and I will close this one